Client/Frontend/Browser/TabLocationView.swift

```diff
@@ -234,10 +280,58 @@ class TabLocationView: UIView {
             attributedString.addAttribute(UIAccessibilitySpeechAttributePitch, value: NSNumber(double: TabLocationViewUX.BaseURLPitch), range: nsRange)
             attributedString.pitchSubstring(baseDomain, withPitch: TabLocationViewUX.HostPitch)
             urlTextField.attributedText = attributedString
+ 
         } else {
             // If we're unable to highlight the domain, just use the URL as is.
             urlTextField.text = url?.absoluteString
         }
+        */
+        
+        // Cliqz: disply only the domain name in the url TextField if possible
+        var urlText = ""
```

Comment:
It would be better to have urlText retrieving logic separated in another function and the styling to have in current function.